book/src/paper-wallet/keypair.md

```diff
@@ -0,0 +1,69 @@
+# Keypair Generation and Recovery
+
+Using the `solana-keygen` tool, it is possible to generate new seed phrases for key derivation as well as recover keypairs directly from a seed phrase and optional passphrase. The seed phrase and passphrase together can be used as a paper wallet.
+
+{% hint style="info" %}
+We do not offer advice on how to securely handle your paper wallet. Please research the security concerns carefully.
+{% endhint %}
+
+## Keypair Generation
+
+Keypair creation can be done using the `solana-keygen new` command. The command will walk you through choosing an optional passphrase and will display the generated seed phrase for your paper wallet.
+
+```bash
+solana-keygen new --no-outfile
+```
+
+{% hint style="warning" %}
+If `--no-outfile` is not specified, the default behavior is to write the keypair to `~/.config/solana/id.json`
+{% endhint %}
+
+Full usage details:
+
+```bash
```

Comment:
Maybe we should just say something like, "See `solana-keygen new --help` for a full list of key generation features"?

I can see this page getting out of sync with the code by the end of the week 🙂